src/dom-manipulation.ts

```diff
@@ -52,13 +52,13 @@ export function addNewGameLine(
 
 /**
  * @description Add indicators to the game
- * @param type the type of indicators
- * @param container the container where to put indicators
- * @param number the number of indicators to add
+ * @param {Indicators} type the type of indicators
+ * @param {HTMLDivElement} container the container where to put indicators
```

Comment:
for the @param is this really interesting to put the type of the parameters before the name of it, on vscode, the first word is colored, Logically the {} indicate that it's the type we talk about but I don't know the result on the IDE

Reply:
see on vscode it's perfect

Reply:
Yeah this is how jsdoc knows the type of the @param, see [this](https://jsdoc.app/tags-param.html) !! :D